README.md

```diff
@@ -244,48 +114,20 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project consider giving it a star on GitHub! Your help reassures me that this initiative is valuable and appreciated.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
-
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank microverse in advance. You helped me to build this project by supporting me and providing requirements.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The file you're referencing does not exist in your directory. Kindly create LICENSE file with the proper content in it and then reference it. That would prevent `404 - page not found` that's experiencing at this moment.

### Your MIT License page
![license](https://github.com/UbdaNam/Vet-Clinic-db-query/assets/26597432/12265a9d-9f94-4586-98a4-bc4a2f8ff097)

Reply:
Thank you @Qoosim for your review 👍. I will make the change and ask for re-review